pkg/ssh/shell/setupMachineMetrics.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+#name:TASK [download monitoring script]
+wget -nd -m https://raw.githubusercontent.com/ava-labs/avalanche-monitoring/main/grafana/monitoring-installer.sh
+#name:TASK [modify permission for monitoring script]
+chmod 755 monitoring-installer.sh
+#name:TASK [set up Prometheus]
+./monitoring-installer.sh --1
```

Comment:
we only need node_exporter in case of dedicated monitoring host, no need to install full prometheus there

Reply:
another thing that we can have one ssh script here and use golang templates if-else case to control what steps go where

Reply:
if we install prometheus, do we start it on each of the monitored nodes ? 
still not clear for me, why prometheus is needed. Node exporter should be able to run standalone